pytest_rts/plugin.py

```diff
@@ -0,0 +1,93 @@
+"""Code for pytest-rts plugin logic"""
+import logging
+import os
+import pytest
+from pytest_rts.pytest.init_phase_plugin import InitPhasePlugin
+from pytest_rts.pytest.normal_phase_plugin import NormalPhasePlugin
+from pytest_rts.pytest.update_phase_plugin import UpdatePhasePlugin
+from pytest_rts.utils.common import update_mapping_db
+from pytest_rts.utils.db import DatabaseHelper, DB_FILE_NAME
+from pytest_rts.select import get_tests_and_data_committed, get_tests_and_data_current
+from pytest_rts.utils.git import get_current_head_hash
+
+
+def pytest_addoption(parser):
+    """Register pytest flags"""
+    parser.addoption("--rts", action="store_true", default=False, help="run rts")
+    parser.addoption(
+        "--rts-init", action="store_true", default=False, help="run rts init"
+    )
+
+
+def pytest_configure(config):
+    """Register RTS plugins based on state"""
+    logger = logging.getLogger()
+    logging.basicConfig(format="%(message)s", level=logging.INFO)
+
+    if config.option.rts_init:
```

Comment:
shall we remove `--rts-init` option and just run init if there is no DB?

Reply:
Possible if wanted. Have to also change the `conftest.py` to initialize the database correctly before every test.

Reply:
in `conftest.py` replacing `subprocess.run(["pytest", "--rts-init"], check=True)` to something like:
* `subprocess.run(["rm", "coverage.db"], check=True)`
* `subprocess.run(["pytest", "--rts"], check=True)`

will do the job if `--rts` checks the existence of the DB